contentPages/migrations/0046_accessibilitystatement.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.20 on 2021-05-18 07:23
```

Comment:
When you have done a whole bunch of migrations for a single story it is good to tidy up at the end. Things look tidier and it is easier to review. If you use the `squashmigrations` command (https://docs.djangoproject.com/en/3.2/topics/migrations/#migration-squashing) it will optimise migration code and put everything in 1 file. 
The command will be something like `python manage.py squashmigrations 0046 0052`